package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@transform-data/transform-react",
-  "version": "1.32.1",
+  "version": "1.33.0",
```

Comment:
I updated this here, but I'm not planning to publish until I've committed other changes related to leveraging this mutation.